pyftpdlib/handlers.py

```diff
@@ -3449,6 +3453,15 @@ def __init__(self, conn, server, ioloop=None):
 
         def __repr__(self):
             return FTPHandler.__repr__(self)
+            
+        #commented out prints - for different Python version support
+        @staticmethod
+        def verify_certs_callback(connection, x509, errnum, errdepth, ok):
+            #if not ok:
+                #print "Bad client certificate detected."
+            #else:
+                #print "Client certificate ok."
+            return ok
```

Comment:
If this is called when a client connects, as I suppose, then it should be a `classmethod`.